src/components/phyloref/PhylorefView.vue

```diff
@@ -51,7 +51,7 @@
               for="definition"
               class="col-form-label col-md-2"
             >
-              Definition in free-form text
+              Definition in free-form text (including qualifiers)
```

Comment:
I thought we were going to leave out the parenthetical text?

Reply:
Yes, I just hadn't gotten around to doing it yet! Fixed in 687dc04.